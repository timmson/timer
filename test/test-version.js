const React = require("react");
const renderer = require("react-test-renderer");

const Version = require("../src/version");

describe("Version should", () => {

	test("return version", () => {
		const component = renderer.create(
			<Version year={"1000"} oldUrl={"old"}/>
		);
		expect(component.toJSON()).toMatchSnapshot();
		component.unmount();
	});
});