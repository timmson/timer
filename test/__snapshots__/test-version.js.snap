// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Version should return version 1`] = `
<div
  className="row"
>
  <div
    className="col"
    style={
      Object {
        "color": "#666",
        "textAlign": "right",
      }
    }
  >
    Версия 
    1000
     года. Ищете 
    <a
      href="old"
    >
      старую версию
    </a>
    ?
  </div>
</div>
`;
